platforms/platform_db/discrete_pcie3.json

```diff
@@ -30,6 +30,7 @@
             "params":
                {
                   "vc-supported": "{ 1, 0, 1, 0 }",
+                  "num-timing-reg-stages": 2,
```

Comment:
Can I override platform's num-timing-reg-stages with that of AFU?

Reply:
No, the idea is to indicate what the platform believes to be needed. You can override the number auto-instantiated by setting num-auto-timing-reg-stages.

Reply:
Ok.